pkg/controller/installation/core_controller.go

```diff
@@ -1800,6 +1800,16 @@ func (r *ReconcileInstallation) setDefaultsOnFelixConfiguration(ctx context.Cont
 			reqLogger.Error(err, "An error occurred when getting the Daemonset resource")
 			return false, err
 		}
+		// Calico-node daemonset is not present. This is a fresh install in eBPF mode.
+		// Set the felix config.
+		if install.Spec.BPFEnabled() {
+			err = setBPFEnabledOnFelixConfiguration(fc, true)
+			if err != nil {
+				reqLogger.Error(err, "Unable to enable eBPF data plane")
```

Comment:
Suggest to update the error message to something like "Unable to enable eBPF data plane with a fresh install".